tests/Rector/LegacyGetterToOcpServerGetRector/config/config.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+use Nextcloud\Rector\Rector\LegacyGetterToOcpServerGetRector;
+use Nextcloud\Rector\ValueObject\LegacyGetterToOcpServerGet;
+use OCP\IRequest;
+use Rector\Config\RectorConfig;
+
+return RectorConfig::configure()
+    ->withConfiguredRule(
+        LegacyGetterToOcpServerGetRector::class,
+        [
+            /** @phpstan-ignore class.notFound */
+            new LegacyGetterToOcpServerGet('getRequest', IRequest::class),
```

Comment:
The above snippet didn't work for me, had to use `new LegacyGetterToOcpServerGet('getRequest', \OCP\IRequest::class),` instead. 

I'm running nextcloud-rector in an own folder and therefore don't have access to the core classes. 

How do you run rector?

Reply:
Did you forget the use `OCP\IRequest;` statement?

Reply:
Likely

Reply:
Yep, I forgot the use statement.